src/main/java/com/uid2/admin/audit/AuditMiddleware.java

```diff
@@ -0,0 +1,30 @@
+package com.uid2.admin.audit;
+
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * AuditMiddleware objects are intended to be attached to any endpoint that the system
+ * wants to keep track of via logging to an external source.
+ */
+public interface AuditMiddleware {
```

Comment:
Do we need interface for middleware? Probably do handler and middlreware in next PR.

Reply:
The purpose of the interface is to maintain flexibility if it turns out we want to pivot away from a QLDB in the future, as it's a possibility that in the future it will be determined that a different DB fits the purposes of this project better.